views/projects.tpl

```diff
@@ -36,23 +36,20 @@
                 "scrollX": true,
                 "dom": "Brtip",
                 "orderClasses": false,
+                "stateSave": true,
```

Comment:
That's all it takes to put it in localstorage?

Reply:
Yes, it's handled magically by the datatables library